### ISSUE TITLE
Add missing aria-label on pages that are missing these

### ISSUE THREAD

**Issue by runeunhjem** [NoroffFEU/agency.noroff.dev#608]
Go through all html pages and add correct and descriptive aria-label to lines that should have these

**Comment by haiderf7** [comment]
Added aria-label to pages/auth/login/index.html

**Comment by haiderf7** [comment]
Added aria-label to pages/listings/listing/create/index.html

**Comment by haiderf7** [comment]
Added aria-label to pages/listings/listing/edit

**Comment by haiderf7** [comment]
Added aria-lable to pages/admin/index.html

**Comment by haiderf7** [comment]
I added aria-lable to all the pages its missing on. 

PR: https://github.com/NoroffFEU/agency.noroff.dev/pull/613

**Comment by haiderf7** [comment]
The requested changes from PR should be done now.

**Comment by haiderf7** [comment]
Fixed last changes as requested now.

**Comment by haiderf7** [comment]
Was some conflicts on the PR because og the prettier update, i tried to run git checkout main -- .prettierrc.json but that didnt resolve the conflict, i spoke with QA and  the QA will resolve the manually this time.
